File: bddtests/src/main/kotlin/mocking/vision/models/PatientNumber.kt
package mocking.vision.models

import javax.xml.bind.annotation.XmlElement

data class PatientNumber(@XmlElement(namespace = "urn:vision")
                         var numberType: String = "NHS",

                         @XmlElement(namespace = "urn:vision")
                         var number: String) {

    constructor() : this("NHS", "")

}

File: xamarinintegrationtests/NHSOnline.HttpMocks/Vision/Models/PatientNumber.cs
using System.Xml.Serialization;

namespace NHSOnline.HttpMocks.Vision.Models
{
    public sealed class PatientNumber
    {
        [XmlElement(ElementName = "numberType", Namespace = "urn:vision")]
        public string? NumberType { get; set; }

        [XmlElement(ElementName = "number", Namespace = "urn:vision")]
        public string? Number { get; set; }
    }
}